mailbox/store/src/test/java/org/apache/james/mailbox/store/StoreMailboxManagerAnnotationTest.java

```diff
@@ -105,46 +110,49 @@ void updateAnnotationsShouldThrowExceptionWhenDoesNotLookupMailbox() {
         when(mailboxMapper.findMailboxByPath(eq(mailboxPath))).thenReturn(Mono.error(new MailboxException()));
 
         assertThatThrownBy(() -> annotationManager.updateAnnotations(mailboxPath, session, ImmutableList.of(PRIVATE_ANNOTATION)))
-            .hasCauseInstanceOf(MailboxException.class);
+            .isInstanceOf(MailboxException.class);
```

Comment:
Why?
- In `MailboxManagerTest`, we assert by `isInstanceOf` API -> we should keep similar for both
- The original of this test is `isInstanceOf`, and after commit 5ccf47c38a651653a79283f9f83b491d72b852e9 (JAMES-3167 Reactify MailboxMapper), we change it to `hasCauseInstanceOf`, So IMO the revert lit is not a big deal